pyproject.toml

```
@@ -0,0 +1,43 @@
+[build-system]
+requires = ["hatchling", "hatch-requirements-txt"]
+build-backend = "hatchling.build"
+
+[project]
+name = "metadata_backend"
+dynamic = ["version", "dependencies"]
+description = "metadata_backend"
+readme = "README.md"
+requires-python = ">=3.11"
+license = {text = "MIT License"}
+authors = [
+    { name = "CSC Developers" },
+]
+classifiers = [
+    "License :: OSI Approved :: MIT License",
```

```suggestion
    "Development Status :: 4 - Beta",
    "Intended Audience :: Developers",
    "Intended Audience :: Information Technology",
    "License :: OSI Approved :: MIT License",
    "Programming Language :: Python :: 3.11",
    "Topic :: Internet :: WWW/HTTP :: HTTP Servers",
```

a few more things here